### PR TITLE
Added support for InputHints (part of the Bot Framework SDK) to control session open / end

### PR DESCRIPTION
**Pull request by garypretty**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,25 @@ actionsOnGoogleAdapter.onUserSignedInHandlerProvider.registerHandler((user) => {
...
+```cs
+var messageText = "Thanks! Can I help you with something else?";
+var messageOptions = new MessageOptions
+            {
+                InputHint = InputHints.ExpectingInput
+            };
+await context.SayAsync(messageText, speakText, options: messageOptions);
+```
+
+By default, if the inputHint for 'expectingInput' is not found, the session will close (i.e. the app.tell method is used when returning a response via the Google Actions SDK), whereas if it is found then the app.ask method is used. If you wish to leave the session open by default, there an environment setting 'DEFAULT_LEAVE_SESSION_OPEN' which, if set to "true", will leave the session open and accept more input by default without needing to specify it explicitly using the inputHint.
```
Comment (daniel-cotton) [inline README.md]: @garypretty Can we change this default to keep the session open by default unless env states otherwise, to avoid a breaking change?

Comment (garypretty) [reply README.md]: Sure, I can do that. I guess I was just thinking about the default behaviour within Bot Framework on a voice channel, that the default is close the conversation unless you explicitly leave it open for further interaction, which when you think about many skills (one hit -> one reply) makes sense.  Do you not think that clearly highlighting the change and asking people to simply set the environment variable would be enough if they want to leave the session open?  Happy either way anyway. 

P.S. I am also going to make one small additional change to look for explicitly closing the conversation using an inputHint if you leave the conversation open by default.

**Comment by garypretty** [comment]
Hi. I have now made the changes you requested to leave the session open by default and also updated the readme to indicate this.

**Comment by daniel-cotton** [comment]
Awesome, thanks @garypretty - merging!

**Pull request merged**